docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
 ## Purpose
+<!--- Replace _Insert_Your_Issue_ID_ placeholder with your Issue i.e. ISSUE-1 -->
+### [_Insert_Your_Issue_ID_](https://github.com/dell/csi-baremetal/issues/_Insert_Your_Issue_ID_)
```

Comment:
How this change is related to other changes?
However, making a full URL link to the issue isn't necessary since it could be achieved with a number sign i.e. #209
Also, it's a good idea to use [linking keywords](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

Reply:
Not related. Just wanted to avoid copying URL each time.

Reply:
@painhardcore please take a look again